R/Achilles.R

```diff
@@ -1006,6 +1029,73 @@ dropAllScratchTables <- function(connectionDetails,
   ParallelLogger::unregisterLogger("dropAllScratchTables")
 }
 
+#' Optimize atlas cache
+#'
+#' @details
+#' Post-processing, optimize data for atlas cache in separate table to help performance.
+#'
+#' @param connectionDetails                An R object of type \code{connectionDetails} created using the function \code{createConnectionDetails} in the \code{DatabaseConnector} package.
+#' @param resultsDatabaseSchema		       Fully qualified name of database schema that we can write final results to. Default is cdmDatabaseSchema.
+#'                                         On SQL Server, this should specifiy both the database and the schema, so for example, on SQL Server, 'cdm_results.dbo'.
+#' @param vocabDatabaseSchema
```

Comment:
Can you fulfill the comment for this parameter:

`#' @param vocabDatabaseSchema		           String name of database schema that contains OMOP Vocabulary. Default is cdmDatabaseSchema. On SQL Server, this should specifiy both the database and the schema, so for example 'results.dbo'.
`

Reply:
fixed